sdk/keyvault/azure-security-keyvault-certificates/vcpkg/Config.cmake.in

```diff
@@ -4,7 +4,8 @@
 @PACKAGE_INIT@
 
 include(CMakeFindDependencyMacro)
-find_dependency(azure-core-cpp "1.3.1")
+find_dependency(azure-core-cpp "1.5.0")
+find_dependency(azure-identity-cpp "1.2.0")
```

Comment:
No client should depend on identity. Core provides all interfaces, Identity provides implementations for them. It is the user app that takes dependency on un uses service client library and identity library, but not that the client library uses identity directly, or depends on it.

Reply:
sounds good  , thank you for the clarification